server/core/src/main/java/dev/slimevr/tracking/processor/skeleton/HumanSkeleton.kt

```diff
@@ -298,11 +298,8 @@ class HumanSkeleton(
 		// Rebuilds the arm skeleton nodes attachments
 		assembleSkeletonArms(true)
 
-		// Refresh headShift
-		humanPoseManager.computeNodeOffset(BoneType.HEAD)
-
-		// Refresh node offsets for arms
-		computeDependentArmOffsets()
```

Comment:
is that method no longer being used? It should be removed if we don't want to use it anymore then

Reply:
It's still used when the arms are switched, I made sure, I figured I'd just leave it because there's no reason to switch that